src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,59 @@
+.calendar {
+  position: relative;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $calendar-gap;
+  width: calc(7 * #{$day-size} + 6 * #{$calendar-gap});
+  margin: 0 auto;
+  padding: 20px $padding;
+
+  &__day {
+    box-sizing: border-box;
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: $day-size;
+    height: $day-size;
+    margin-left: 0 * $day-size + $calendar-gap * 0;
+    border: 1px solid black;
+    background-color: #eee;
+    font-family: "Arial", "Roboto", sans-serif;
+    font-size: 30px;
+
+    &:hover {
+      cursor: pointer;
+      background-color: pink;
+      transform: translateY(-20px);
+      transition: transform 0.5s;
```

Comment:
transition shouldn't be on hover, move it in the &__day seelctor, in this way transition will works in both sides